frontend/tests/controllers-test/analyticsCtrl.test.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
```

Comment:
Can you indent this file? Would be ideal to have all files you commit to have them indented. At least the ones(files or part of chunks of code) being newly added by you.

Reply:
@galipremsagar It's already indented correctly but not showing same as in my editor.